### ISSUE TITLE
Bundle type declarations (through option?)

### ISSUE THREAD

**Issue by tunnckoCore** [zeit/ncc#482]
Probably using https://www.npmjs.com/package/dts-bundle-webpack?

If it's an option it can be run after the main build. 

```
ncc build src/index.ts -o dist/main
ncc build dist/types/index.d.ts -o dist/typings --dts
rm -rf dist/types
```

Would be good if it reads the `typings` / `types` field.

**Comment by cinderblock** [comment]
If I remember correctly, all you need to do is to add `"declaration": true,` to your `tsconfig.json`.

**Comment by tunnckoCore** [comment]
Nope, I don't think so. :) I wouldn't open issue just for the sake of opening it.
